src/docs/transactions.dox

```diff
@@ -148,4 +148,27 @@ WT_SESSION::snapshot with a configuration that includes
 the semantics supported by the drop configuration.
 
 Named snapshots are not durable: they do not survive WT_CONNECTION::close.
+
+@section transaction_timestamps Timestamps
+
+Applications can assign transactions explicit commit timestamps, then read
+as-of a timestamp.  Timestamps are communicated to WiredTiger as hexadecimal
+values with a length configurable at build time (default 8 bytes).
```

Comment:
If the application's timestamp size is 19 bytes, it should configure a timestamp size of 19.  The encoding would be up to 38 bytes of hex plus a NUL byte.